browser-test/utils.js

```diff
@@ -0,0 +1,39 @@
+import {exec} from "child_process";
+import testData from "./testData.json";
+
+const waitTime = 20000;
```

Comment:
why 20 secs? Looks too much to me. My tests were failing because env issues and it took too long to know what's going on.